x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/policy_template_selectors.tsx

```diff
@@ -82,6 +96,34 @@ export const PolicyTemplateInfo = ({ postureType }: PolicyTemplateInfoProps) =>
         defaultMessage="Select the cloud service provider (CSP) you want to monitor and then fill in the name and description to help identify this integration"
       />
     )}
+    {postureType === VULN_MGMT_POLICY_TEMPLATE && (
+      <>
+        <EuiCallOut
+          iconType="iInCircle"
+          color="primary"
+          title={
+            <FormattedMessage
+              id="xpack.csp.fleetIntegration.cnvm.additionalChargesCalloutTitle"
+              defaultMessage="Additional charges on cloud provider billing account."
```

Comment:
design suggests this without a period `Additional charges on cloud provider billing account`